spec/system/provider_interface/provider_manages_application_interviews_spec.rb

```diff
@@ -145,6 +149,18 @@ def and_an_interview_has_been_created(date)
 
   alias_method :then_another_interview_has_been_created, :and_an_interview_has_been_created
 
+  def when_i_navigate_to_notes_tab
+    click_on 'Notes'
+  end
+
+  def and_i_do_not_see_the_set_up_interview_button
+    expect(page).not_to have_content 'Set up interview'
```

Comment:
Would the `have_button` or `have_link` matcher be more specific here?

Reply:
👍